### DISCUSSION TITLE
Is there a way to rotate the map?

### DISCUSSION THREAD

**Discussion by modo1302** [humbertogontijo/homeassistant-roborock#401] [Q&A] [answered]
Hi, was looking through the documentation and didn't find the asnwer - is there any way to rotate the map? :)

**Comment by humbertogontijo** [comment, marked as answer]
Yes, under integration configuration > camera > rotation

**Reply by modo1302** [reply]
thank you!

**Comment by nerijus** [comment]
I cannot find configure button for the integration in HA 202312 version.
